src/custom-types.json

```diff
@@ -583,6 +583,13 @@
       "http://facebook.github.io/react/docs/getting-started.html"
     ]
   },
+  "text/vue": {
+    "compressible": true,
+    "extensions": ["vue"],
+    "sources": [
+      "http://vuejs.org/v2/guide/single-file-components.html"
```

Comment:
> There's nothing about text/jsx on the page http://facebook.github.io/react/docs/getting-started.html either.
 
It looks like their site has been re-arranged. You can see that it now is redirecting to `hello-world.html`. Here is what the page used to look like: https://shripadk.github.io/react/docs/getting-started.html